klog.go

```diff
@@ -699,7 +699,11 @@ func (buf *buffer) someDigits(i, d int) int {
 }
 
 func (l *loggingT) println(s severity, logger *logr.Logger, filter LogFilter, args ...interface{}) {
-	buf, file, line := l.header(s, 0)
+	l.printlnDepth(s, logger, filter, 1, args...)
+}
+
+func (l *loggingT) printlnDepth(s severity, logger *logr.Logger, filter LogFilter, depth int, args ...interface{}) {
```

Comment:
What is suspicious here is that depth is not passed through to `l.output`. Wrapping `printlnDepth` with `println` and keeping the same constant 0 as before can't be right.

Reply:
I assume it should be something like depth+1

Reply:
Pasing depth through fixed it for me -> https://github.com/kubernetes/klog/pull/287

Reply:
Whether it is "1" or "depth + 1" depends a lot on the actual semantic, which isn't well defined. This is almost impossible to get right without tests :cry: 

In this case one can reason like this: 0 was correct before, now we add one layer and pass depth=1 -> passing depth is right, depth+1 would be one level too many. Testing with the JSON tests in Kubernetes confirms that.